virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -257,4 +257,12 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 +
 Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to FIPS servers running on RHEL 9. Similarly, RHEL 9 clients in FIPS mode cannot connect to servers that only support TLS 1.2 without EMS. This in practice means that these clients cannot connect to servers on RHEL 6, RHEL 7 and non-RHEL legacy operating systems. This is because the legacy 1.0.x versions of OpenSSL do not support EMS or TLS 1.3. For more information, see link:https://access.redhat.com/solutions/7018256[TLS Extension "Extended Master Secret" enforced with Red Hat Enterprise Linux 9.2].
 
-** As a workaround, update legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
\ No newline at end of file
+** As a workaround, update legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
+
+
+[discrete]
+[id="virt-4-15-ki-webconsole"]
+==== Web console
+//CNV-38293
+* When you first deploy an {product-title} cluster, creating VMs from templates or instance types, by using the web console, fails if you do not have `cluster-admin` permissions.
```

Comment:
```suggestion
* When you first deploy an {product-title} cluster, creating VMs from templates or instance types by using the web console, fails if you do not have `cluster-admin` permissions.
```

Reply:
Only one small nit: recommend omitting the comma, it splits the condition in half and makes it difficult to read.